public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -399,16 +404,16 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
     if (this.queryTimeout) {
       data['timeout'] = this.queryTimeout;
     }
-    return this._request(url, data, { requestId: query.requestId, headers: query.headers }).catch((err: any) =>
-      this.handleErrors(err, query)
-    );
+    return this._request(url, data, { requestId: query.requestId, headers: query.headers }).catch((err: any) => {
+      if (err.cancelled) {
```

Comment:
Small question, why is this if statement moved outside the handleErrors function?
```
if (err.cancelled) {
```

Reply:
I think my thinking was that I wanted it to match how things were done in https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/loki/datasource.ts#L124 but I can change it if that's not necessary

Reply:
Ok, thanks!